mantle/fcos/metadata.go

```diff
@@ -68,7 +69,87 @@ type ReleaseCommit struct {
 	Checksum     string `json:"checksum"`
 }
 
-func FetchAndParseReleaseIndex(url string) (*ReleaseIndex, error) {
+// This models stream metadata:
+// https://github.com/coreos/fedora-coreos-tracker/tree/master/metadata/stream
```

Comment:
Cool, and this will help with adding
`kola spawn -p aws --fcos-stream stable` etc.

Reply:
Though one other side note: Did you consider generating this from a schema too?

Reply:
For now, I just copied this over from fedora-coreos-stream-generator.  But yeah, that'd be a good thing to do.